### DISCUSSION TITLE
Possibility to override the limit

### DISCUSSION THREAD

**Discussion by bjora857** [gtmassey/Analytics#42] [Ideas]
It would be nice to be able to override the default limit of 10 000 rows returned. So something like:

`Analytics::query()->setLimit(50000);`

**Comment by Plytas** [comment]
Good suggestion! I opened a PR https://github.com/gtmassey/laravel-analytics/pull/43 to support both `limit()` and `offset()`.

**Reply by bjora857** [reply]
Perfect. Much appricated

**Reply by gtmassey** [reply]
Hi! Thanks for the suggestion @bjora857, and thanks @Plytas for the implementation! I'm going to review the PR after the new year and merge then. I'm taking some time away from code for a bit!

**Reply by bjora857** [reply]
That sounds like a good plan. Enjoy your time away from the code @gtmassey. 
And thanks for putting together this package!

**Reply by gtmassey** [reply]
v2.1.0 is out now! It includes the limits and offsets, along with the filters from another PR by @Plytas. 

Enjoy! Thanks for the suggestion and thanks @Plytas for the work!
